### ISSUE TITLE
Copper dupe with ammo press

### ISSUE THREAD

**Issue by k00lk4t** [Cubed-Development/Modern-Warfare-Cubed#420]
### Describe the bug

As seen in the video, the crafting recipe for the ammo is 6cu+6gp and it makes 6 ammo, but when dismantling the same ammo the ammo press gives you 6cu+6gp _**per**_ ammo, leading to a crazy amount of Copper and Gunpowder.

### Reproducibility

- [X] I reproduced this issue with as few other mods as possible installed.
- [ ] I am unable to reproduce this issue consistently.

### Screenshots and/or videos

https://cdn.discordapp.com/attachments/715601459076661369/1227688658648174622/2024-04-10_20-35-46.mp4?ex=66295129&is=6616dc29&hm=9a7d644dea00f53914135b541cb8c1bf5cc7e0115f77e9cb4215e45935611e0a&

### Logs

not avaliable

### MWC Version

0.1.5

### Forge Version

latest

### Red Core Version

0.5.1

### MixinBooter Version

8.9

### Java Version

1.8 - 392

### Java Virtual Machine

Hotspot

### Operating System

Windows 10

### GPU Vendor

NVIDIA

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by Paneedah** [comment]
Yeahhhhhh... easy fix - I'll give it a hotfix later.

**Comment by k00lk4t** [comment]
Thanks for the quick response! While i'm here, is there any work on Unidict support for the mod?

**Comment by Desoroxxx** [comment]
This is the 420 issue? Nice

**Comment by k00lk4t** [comment]
Hello, has this been fixed yet?

**Comment by Desoroxxx** [comment]
> Hello, has this been fixed yet?

No, but I have found a solution that I am satisfied with that should come very soon in a revamp of the crafting system

**Comment by strubium** [comment]
Related: as reported by @En3rgyZ3d. 

"Placing an hopper under an ammo press while it has bullets in its output inventory, dupes the bullet.
For example, if we have 60 bullets in the output and you place an hopper, it will take first 60 bullets, then 59, then 58... until 0, greatly increasing the number of bullets."
